src/packagedcode/windows_helpers.py

```diff
@@ -0,0 +1,311 @@
+import os
+from pathlib import PureWindowsPath
+
+import attr
+import logbook
+from regipy.exceptions import NoRegistrySubkeysException
```

Comment:
If you split the registry code in a separate module (or not) ... it would make sense to have regipy as an optional extra dep and guard at least the imports inside the functions using them or with a try/except to ensure that the module always imports at the minimum.